MSAL/src/native_auth/public/state_machine/error/SignInAfterSignUpError.swift

```diff
@@ -26,11 +26,16 @@ import Foundation
 
 @objc
 public class SignInAfterSignUpError: MSALNativeAuthError {
+    /// Describes an error that provides messages describing why an error occurred and provides more information about the error.
     public override var errorDescription: String? {
         if let description = super.errorDescription {
             return description
         }
 
         return "General error"
     }
+
+    init(message: String? = nil) {
+        super.init(identifier: "Sign in after sign up", message: message)
```

Comment:
I understand there is no need for a `type` member for some of the error subclasses like this one because there is only a single type, but for the sake of consistency I think it would be good to follow the same pattern for all error types.

Reply:
Ok, added.